airbyte-integrations/connectors/source-talkdesk-explore/source_talkdesk_explore/spec.json

```diff
@@ -23,20 +23,29 @@
         "default": "UTC",
         "order": 1
       },
+      "region": {
+        "type": "string",
+        "title": "REGION",
+        "description": "Region to use when generating reports. Only US, Canada and Europe are supported. Defaults to US region.",
+        "examples": ["US", "Canada", "Europe" ],
+        "pattern": "^(US|Canada|Europe)$",
+        "default": "US",
+        "order": 2
```

Comment:
You can use `enum` to have a better user experience, check other connectors spec. I don't have one example but let me know if you need help here.

Reply:
Done in https://github.com/airbytehq/airbyte/pull/17587/commits/a7df72dc59e2e69f38eb3d21c15e71e48e07e57e and https://github.com/airbytehq/airbyte/pull/17587/commits/731df211ca966b64751cd50897a284afe9c7dec6